firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/ErrorMessages.java

```diff
@@ -28,8 +30,12 @@ class ErrorMessages {
 
   static final String AUTHENTICATION_CANCELED = "Tester canceled the authentication flow.";
 
-  static final String NOT_FOUND_ERROR =
-      "Release not found. An update was not available for the current tester and app. Make sure that FirebaseAppDistribution#checkForNewRelease returns with a non-null  AppDistributionRelease before calling FirebaseAppDistribution#updateApp";
+  static final String RELEASE_NOT_FOUND_ERROR =
+      "Release not found. An update was not available for the current tester and app. Make sure that FirebaseAppDistribution#checkForNewRelease returns with a non-null AppDistributionRelease before calling FirebaseAppDistribution#updateApp";
```

Comment:
nit: `FirebaseAppDistribution.checkForNewRelease()` and `FirebaseAppDistribution.updateApp()` are more legible - this is not JavaDoc

Reply:
I actually like `checkForNewRelease()` and `updateApp()`!